compiler/damlc/tests/daml-test-files/InterfaceDesugared.daml

```diff
@@ -3,6 +3,7 @@
 
 -- @SINCE-LF-FEATURE DAML_INTERFACE
 -- @WARN Modules compiled with the DatatypeContexts language extension
+-- @ERROR InterfaceDesugared:Asset does not implement interface InterfaceDesugared:Token
```

Comment:
since the desugared file cannot contain the definition for `$implements_Token_Asset` (it would be a parse error), we get this error.

Reply:
should no longer be an issue :)

Reply:
indeed! thanks for the reminder